### DISCUSSION TITLE
Just to be sure that we are talking about height above radar and not altitude above mean sea level

### DISCUSSION THREAD

**Discussion by johnnypeste** [ARM-DOE/pyart#1688] [General]
Hi,

I'm using a radar which is 500 m above mean sea level.

I wanto to use:

grid = pyart.map.grid_from_radars(radar,
                                  gridding_algo='map_gates_to_grid',
                                  roi_func="dist_beam",
                                  constant_roi=None,
                                  grid_shape=(z_grid_points,
                                              y_grid_points,
                                              x_grid_points),
                                  grid_limits=(z_grid_limits,
                                               y_grid_limits,
                                               x_grid_limits),
                                 )

to be able to build a cappi at 1000m and 2000m above mean sea level.

z_grid_limits should be:

(500 , 1500)

or:

(1000 , 2000)

??

I think it's the first option, but I'm not sure...

**Comment by mgrover1** [comment]
The height refers to height above the radar, so (500, 1500)
